src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/TelemetryClientService.kt

```diff
@@ -228,7 +228,7 @@ class TelemetryClientService(
     val reportEvent = mutableMapOf<String, String>()
     with(overbookedSession) {
       reportEvent["sessionDate"] = formatDateToString(sessionDate)
-      reportEvent["prisonCode"] = prisonCode
+      reportEvent["prisonId"] = prisonCode
```

Comment:
changed prisonCode to prisonId as suggested.

Reply:
As discussed, although it's not technically the prisonId, we use this across the telemetry events and it's a quick fix. Long term these should be moved to be called prisonCode.